test/run_pass/test_fs_write_stream.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var assert = require('assert');
+var common = require('../common');
+
+var dstFilePath = path.join(__dirname, '../resources/test2.txt');
+var originalContents = fs.readFileSync(dstFilePath);
+var buff1 = Buffer.from('IoT');
+var buff2 = Buffer.from('.js');
+
+function nop() {}
+
+var stream = fs.createWriteStream(dstFilePath);
+stream.write(buff1, nop);
+stream.write(buff2, nop);
+stream.on('finish', common.mustCall(function onend() {
```

Comment:
```suggestion
stream.on('finish', common.mustCall(function onfinish() {
```
Should we align the event listener name and the event name?